examples/coverage_test/src/lib.rs

```diff
@@ -0,0 +1,374 @@
+// Import hacspec and all needed definitions.
```

Comment:
How about doing this as an actual test in the hacspec-lib crate? Then we can get actual code coverage tools to tell us how good it is (grcov, tarpaulin etc.).

Reply:
The main idea was to test coverage of the implementation for the backends, however I would be nice to make sure it is actually a good test, by checking code coverage.

Reply:
Yeah both would indeed be nice.